src/canvas.tsx

```diff
@@ -605,15 +605,15 @@ export const useCanvas = (props: UseCanvasProps): DomEventHandlers => {
       gl.toneMapping = THREE.ACESFilmicToneMapping
       gl.outputEncoding = THREE.sRGBEncoding
     }
-  }, [shadowMap, colorManagement])
+  }, [shadowMap, colorManagement]) // React Hook React.useLayoutEffect has a missing dependency: 'gl'
```

Comment:
here is possible missing hook dependency. please clarify intention

Reply:
you can add gl